packages/malloy-db-snowflake/package.json

```diff
@@ -21,7 +21,7 @@
     "prepublishOnly": "npm run build"
   },
   "dependencies": {
-    "@malloydata/malloy": "^0.0.99",
+    "@malloydata/malloy": "file:../malloy",
```

Comment:
revert?

Reply:
yep this needs to be removed. though we need to think how we will deploy this to npm